goconf/conferences/views.py

```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render
+from django.utils import timezone
+from .models import Conf
+
+
+def conf_list(request):
```

Comment:
Please use `ListView` from Django.
Function views are so 2010. They are fast and easy, nice to show in a tutorial, but class based views rock. You can do this view in two lines:
```
class ConferenceList(ListView):
    model = Conference
```
You can read more about it here: https://docs.djangoproject.com/en/2.1/topics/class-based-views/generic-display/#generic-views-of-objects